config/config.properties

```diff
@@ -349,6 +349,20 @@ extendeds3.url=localhost:9020
 # Recommended values: a path to a locally mounted directory that sits on top of a fast SSD.
 #rocksdb.dbDir=/tmp/pravega/cache
 
+# Parameters to bound the memory consumption of RocksDB. The total memory consumption of the database is mostly dictated
+# by writeBufferSizeMB (memtable) and readCacheSizeMB (block cache). However, RocksDB also stores data in memory related
+# to internal indexes in addition to the previous parameters (e.g., it may range between 5% to 30% of the total memory
+# consumption depending on the configuration and data at hand). The size of the internal indexes in RocksDB mainly
+# depend on the size of data blocks in the cache (cacheBlockSizeKB). That is, if you increase cacheBlockSizeKB, the
+# number of blocks will decrease, so the index size will also reduce linearly. For an explanation in depth of these
+# parameters, we refer to: https://github.com/facebook/rocksdb/wiki/Memory-usage-in-RocksDB.
+
+#rocksdb.writeBufferSizeMB=64
```

Comment:
It would be really helpful to a sys admin if each of these config settings has its own individual description. What I've done throughout this file was to simply copy the Javadoc from each of these here, and add a Recommended value to each.

This is in addition to the blob of text you added just above this.

Reply:
Thanks, I added the description of each parameter with the comment available in `RocksDBConfig.java`. Moreover, I think that a "recommended value" is somewhat complex in this field that depends on various aspects. For this reason, I will create a Wiki page in the Pravega repository (`https://github.com/pravega/pravega/wiki/RocksDB-Configuration`) including experiments and guidelines to users for configuring RocksDB depending on their needs/resources. Does this sound reasonable?